### PR TITLE
notebookbar: remove menubartoolitems #8029

### LINKED ISSUE #8029: Replace all menubartoolitems with customtoolitems
**This is an [Easy Hack](https://collaboraonline.github.io/post/easyhacks/).**
Potential mentors: @eszkadev

### Detailed description and rationale
There were in the past initial approach to share code between menubar and notebookbar by using `(big)menubartoolitem` element in the notebookbar JSON. Now we have better widget for that: `(big)customtoolitem` that will not use Menubar code at all so reduces external dependencies for us (to smartmenus library).

### Code pointers
- use `git grep menubartoolitem` command to identify existing items 
- we want to move implementation of command/action from menubar (https://github.com/CollaboraOnline/online/blob/master/browser/src/control/Control.Menubar.js#L1841) to `dispatch` function https://github.com/CollaboraOnline/online/blob/master/browser/src/control/Toolbar.js#L945
- `customtoolitem` runs `command` using `map.dispatch(command)` function where we need to put command implementation
- adjust Menubar code to also use `map.dispatch(command)` for item you moved
- JSON with notebookbar description is in `https://github.com/CollaboraOnline/online/blob/master/browser/src/control/Control.NotebookbarWriter.js` for Writer and similar files for other apps

### PR DESCRIPTION
**Pull request by eszkadev**
Fixes #8029 
All menubartoolitems which left are now converted to customtoolitems which use dispatcher to invoke internal commands. Code is shared with menubar this way. Remove it so noone will add more of them again.

### CODE REVIEW

**Comment by eszkadev** [comment]
still WIP

File: browser/src/control/Control.NotebookbarBuilder.js [hunk 1]
```diff
@@ -477,6 +436,17 @@ L.Control.NotebookbarBuilder = L.Control.JSDialogBuilder.extend({
...
 				}
 			] : []);
+		} else if (docType === 'drawing') {
+			submenuOpts = [
+				{
+					'action': 'downloadas-odg',
+					'text': _('ODF Drawing (.odg)')
+				},
+				{
+					'action': 'downloadas-png',
```
Comment (eszkadev) [inline browser/src/control/Control.NotebookbarBuilder.js:446]: I noticed we don't have icon for download png

Comment (Darshan-upadhyay1110) [reply browser/src/control/Control.NotebookbarBuilder.js:446]: Yes need to add this in notebookbar.css
```
[class*='as-png-submenu-icon'] {
	background: url('images/lc_insertgraphic.svg') center;
}
```

**Review by Darshan-upadhyay1110 — APPROVED** [review]
All changes have been tested. & All works good.

Note: Just a missing icon which I already indicated in a comment thread, otherwise all looks good to me

Thanks , @eszkadev

**Comment by Darshan-upadhyay1110** [comment]
https://github.com/CollaboraOnline/online/pull/9886

**Pull request merged**